src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -495,10 +513,15 @@ private void ValidateUploadOptions(MigrateRepoCommandArgs args)
             {
                 throw new OctoshiftCliException("Either --aws-secret-key or AWS_SECRET_KEY environment variable must be set.");
             }
+
+            if (!GetAwsRegion(args).HasValue())
```

Comment:
Can we add a validation that requires the user to set their AWS region if they provide a session token? We're only trying to preserve backwards compatibility for old scripts really, and if you specify a session token, then it's a good time to force the region too I think.

Reply:
Addressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc